### PR TITLE
Create Executive Vote - April 8, 2022.md

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
First Exec draft for Friday @blimpa @LongForWisdom

### CODE REVIEW

File: governance/votes/Executive Vote - April 8, 2022.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Parameter Changes from the latest [proposal](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-31-march-2022/14347) from the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355) including [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee), [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line), [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and Aave D3M [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar), listed in detail below.
+- An increase in the CRVV1ETHSTETH-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line), detailed below.
+- A total of 77,183 DAI will be paid to nine Recognized Delegates as per [MIP61 Recognized Delegate Compensation](https://mips.makerdao.com/mips/details/MIP61), detailed below.
```
Comment (blimpa) [inline governance/votes/Executive Vote - April 8, 2022.md:21]: (In the previous exec we bolded the DAI amounts under the Executive Summary too)

File: governance/votes/Executive Vote - April 8, 2022.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- 50,000 DAI will be transferred to the ETHAmsterdam Event SPF multi-sig as a result of their successful [Special Purpose Fund (SPF) application](https://mips.makerdao.com/mips/details/MIP55c3SP3), detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
```
Comment (blimpa) [inline governance/votes/Executive Vote - April 8, 2022.md:24]: No additions, I think

File: governance/votes/Executive Vote - April 8, 2022.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Parameter Changes from the latest [proposal](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-31-march-2022/14347) from the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355) including [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee), [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line), [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and Aave D3M [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar), listed in detail below.
+- An increase in the CRVV1ETHSTETH-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line), detailed below.
```
Comment (blimpa) [inline governance/votes/Executive Vote - April 8, 2022.md:20]: This is the only item in the Executive Summary for which we're not giving a source post/doc

File: governance/votes/Executive Vote - April 8, 2022.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+**[Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Increase**
+- Increase the **GUNIV3DAIUSDC1-A** Target Available Debt by 40 million DAI from **10 million DAI** to **50 million DAI**.
+
+**D3M [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar) Increase**
+- Increase the **DIRECT-AAVEV2-DAI** Target Borrow Rate by 0.65% from **2.85%** to **3.5%**.
+
+### CRVV1ETHSTETH-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increase
+
+As a result of this [post](https://forum.makerdao.com/t/immediate-short-term-parameter-changes-proposal-for-crvv1ethsteth-a-dc-and-gap-increase/14476) from the [Risk Core Unit (RISK-001)](https://mips.makerdao.com/mips/details/MIP39c2SP2) the CRVV1ETHSTETH-A Maximum Debt Ceiling will be increased by 2 million DAI from **3 million DAI** to **5 million DAI** if this executive proposal passes.
```
Comment (blimpa) [inline governance/votes/Executive Vote - April 8, 2022.md:58]: "As a result [...] (RISK-001)**,** the"

File: governance/votes/Executive Vote - April 8, 2022.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+* 6,606 DAI transferred to [GFX Labs](https://vote.makerdao.com/address/0xf60d7a62c98f65480725255e831de531efe3fe14) at [0xa6e8772af29b29B9202a073f8E36f447689BEef6](https://etherscan.io/address/0xa6e8772af29b29B9202a073f8E36f447689BEef6).
+* 283 DAI transferred to [Doo](https://vote.makerdao.com/address/0x8804d391472126da56b9a560aef6c6d5aaa7607b#delegate-credentials) at [0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0](https://etherscan.io/address/0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0).
+
+### ETHAmsterdam Event Special Purpose Fund DAI Distribution
+
+As per their successful [Special Purpose Fund](https://mips.makerdao.com/mips/details/MIP55) (SPF) [application](https://mips.makerdao.com/mips/details/MIP55c3SP3), 50,000 DAI will be transferred to the ETHAmsterdam SPF Event multi-sig wallet at the address [0xF34ac684BA2734039772f0C0d77bc2545e819212](https://etherscan.io/address/0xF34ac684BA2734039772f0C0d77bc2545e819212) if this executive proposal passes.
```
Comment (blimpa) [inline governance/votes/Executive Vote - April 8, 2022.md:78]: Their? Whose?

Comment (blimpa) [reply governance/votes/Executive Vote - April 8, 2022.md:78]: Oh, "their" is also used in the Executive Summary. Not sure how clear that is

**Review by blimpa — CHANGES_REQUESTED** [review]

File: governance/votes/Executive Vote - April 8, 2022.md (hunk 5 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive Vote - April 8, 2022.md:78]: Changed "their" to "this".

**Review by blimpa — APPROVED** [review]
LGTM

**Review by LongForWisdom — APPROVED** [review]
LGTM

**Pull request merged**